frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeListView.tsx

```diff
@@ -24,6 +24,8 @@ const CustomServingRuntimeListView: React.FC = () => {
   const navigate = useNavigate();
 
   const [deleteTemplate, setDeleteTemplate] = React.useState<TemplateKind>();
+
+  // TODO: add search functions
```

Comment:
Don't add the TODO here, let's just control that with tickets, im gonna start a conversation to track this cause im not sure if we want that.